bin/compile

```diff
@@ -211,6 +211,17 @@ cache_build() {
 header "Caching build"
 cache_build | output "$LOG_FILE"
 
+prune_devdependencies() {
+  if $YARN; then
+    yarn_prune_devdependencies "$BUILD_DIR"
+  else
+    npm_prune_devdependencies "$BUILD_DIR"
+  fi
+}
+
+header "Pruning devDependencies"
```

Comment:
This is the terminology that npm uses, but I don't love it.

Is there a clearer way to get across "we're removing the modules defined in `devDepenencies` from `node_modules`"?

Reply:
If it's the command in both `yarn` and `npm`, it's probably fine.